test/e2e/config/e2e_conf.yaml

```diff
@@ -18,6 +18,15 @@ providers:
 - name: cluster-api
   type: CoreProvider
   versions:
+  - name: v0.4.7
```

Comment:
I do not think it is recommended to use a var for that for two reasons:
- It may lead to a conflict between the var value and what we expect in this file
- The name does not necessarily need to be the exact version value